.github/workflows/build.yml

```diff
@@ -0,0 +1,37 @@
+name: Nix Flake Build
+
+on:
+  pull_request:
+    branches:
```

Comment:
We may want to run test builds on all branches?

Reply:
yeah i forgot that